src/theory/bv/bitblast/eager_bitblaster.cpp

```diff
@@ -262,6 +265,35 @@ bool EagerBitblaster::collectModelInfo(TheoryModel* m, bool fullModel)
       }
     }
   }
+
+  // Collect the values for the Boolean variables
+  std::vector<TNode> vars;
+  d_cnfStream->getBooleanVariables(vars);
+  for (TNode var : vars)
+  {
+    Node const_value;
+
+    if (d_cnfStream->hasLiteral(var))
+    {
+      prop::SatLiteral bit = d_cnfStream->getLiteral(var);
+      prop::SatValue bit_value = d_satSolver->value(bit);
+      Assert(bit_value != prop::SAT_VALUE_UNKNOWN);
+      const_value = nm->mkConst(bit_value == prop::SAT_VALUE_TRUE);
+    }
+    else
```

Comment:
This case should never happen, right?

Reply:
Yeah, as discussed offline, this probably shouldn't be the case. I removed it and replaced it with an assertion.